.github/workflows/deploy.yml

```diff
@@ -136,6 +182,20 @@ jobs:
           dockerfiles: |
             ./docker/Dockerfile-ubi
 
+
+      - name: Push to local
+        id: push-local
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }}
+          registry: localhost:5000
+          tls-verify: false
+
+      - name: Clair Scan
+        run: |
+          ./.github/clair-scan.sh medium localhost:5000/${{ steps.build-image.outputs.image }}:${{ env.IMAGE_TAG_PRE }}
+
       - name: Push To quay.io
```

Comment:
how can you use Quay?

Reply:
Sorry, I dont understand the question. With this action?